woocommerce/woocommerce
# [Accessibility] Add endpoint title to the document title

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #43632 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the My Account page on the frontend.
2. Verify the document title is "My account – {Store name}".
    Example:
    ![image](https://github.com/user-attachments/assets/b4b65196-936c-4c8b-8735-3ce232c060a3)

3. Go to each one of the My account sub-pages (except "Dashboard" and "Log out" ones) and verify the document title changes to "{Sub-page} – {Store name}". 
    Example
    - "Orders – {Store name}"
    - "Download – {Store name}"
    - "Addresses – {Store name}"
    - "Account details – {Store name}"



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
